### PR TITLE
typo fixes - https://github.com/vlajos/misspell_fixer

### PR DESCRIPTION
**Pull request by vlajos**

### CODE REVIEW

**Comment by arschmitz** [comment]
@vlajos can you please sign our CLA http://contribute.jquery.org/CLA/ and then I will look at this

**Comment by vlajos** [comment]
I just signed the CLA.

**Comment by arschmitz** [comment]
@vlajos thank you!

File: tests/unit/init/weird_file_name_core.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ $( document ).on( "mobileinit", function() {
 	// one call to itself
 	$.mobile.initializePage = function() {
 
-		$.mobile.intializePage = origInitializePage;
+		$.mobile.initializePage = origInitializePage;
```
Comment (arschmitz) [inline tests/unit/init/weird_file_name_core.js]: This needs to be reverted, then i think every thing here is fine this corresponds to an option in the actual code.

**Comment by vlajos** [comment]
I have just reverted that line. (And signed the CLA again typing my name as it is in the commit.)

**Pull request closed without merging**